sqlx-core/src/mysql/connection/executor.rs

```diff
@@ -275,10 +275,10 @@ impl<'c> Executor<'c> for &'c mut MySqlConnection {
                 for _ in 0..(ok.columns as usize) {
                     let def: ColumnDefinition = self.stream.recv().await?;
                     let ty = MySqlTypeInfo::from_column(&def);
-                    let name = def.name()?;
+                    let name = def.alias()?;
```

Comment:
This variable should get renamed to `alias` for clarity.

Reply:
Fixed :)